config/jobs/kubernetes-sigs/kueue/kueue-release-blocking.yaml

```diff
@@ -0,0 +1,253 @@
+      testgrid-alert-email: release-team@kubernetes.io
+      testgrid-num-columns-recent: '6'
+      testgrid-num-failures-to-alert: "1"
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: kueue
+        base_ref: master
+        path_alias: kubernetes-sigs/kueue
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230222-b5208facd4-master
+          env:
+            - name: E2E_KIND_VERSION
+              value: kindest/node:v1.24.7
+          command:
+            # generic runner script, handles DIND, bazelrc for caching, etc.
+            - runner.sh
+          args:
+            - make
+            - kind-image-build
+            - test-e2e
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
+  - interval: 1h
+    name: periodic-kueue-test-e2e-main-1-25
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-kueue-test-e2e-main-1-25
+      description: "Run kueue end to end tests for Kubernetes 1.25"
+      testgrid-alert-email: release-team@kubernetes.io
+      testgrid-num-columns-recent: '6'
+      testgrid-num-failures-to-alert: "1"
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: kueue
+        base_ref: master
+        path_alias: kubernetes-sigs/kueue
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230222-b5208facd4-master
+          env:
+            - name: E2E_KIND_VERSION
+              value: kindest/node:v1.25.3
+          command:
+            # generic runner script, handles DIND, bazelrc for caching, etc.
+            - runner.sh
+          args:
+            - make
+            - kind-image-build
+            - test-e2e
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
+  - interval: 1h
+    name: periodic-kueue-verify-main
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-kueue-verify-main
+      description: "Run kueue verify checks"
+      testgrid-alert-email: release-team@kubernetes.io
+      testgrid-num-columns-recent: '6'
+      testgrid-num-failures-to-alert: "1"
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: kueue
+        base_ref: master
+        path_alias: kubernetes-sigs/kueue
+    decorate: true
+    decoration_config:
+      timeout: 1h
+    spec:
+      containers:
+        - image: golang:1.19
+          command:
+            - make
+          args:
+            - verify
+  - interval: 1h
```

Comment:
Maybe we can get rid of this (also for the presubmits). The E2E tests already do a build.

Reply:
Also, follow this discussion https://github.com/kubernetes/test-infra/pull/27411#issuecomment-1466139982

@tenzen-y

Reply:
I replied to the above PR.